inspector/src/components/actionTabs/tabs/propertyGrids/materials/textures/defaultTools/eyedropper.ts

```diff
@@ -29,15 +29,16 @@ export const Eyedropper : IToolData = {
         setup () {
             this.pointerObserver = this.getParameters().scene.onPointerObservable.add((pointerInfo) => {
                 if (pointerInfo.pickInfo?.hit) {
-                    if (pointerInfo.type === PointerEventTypes.POINTERDOWN) {
+                    if (pointerInfo.type === PointerEventTypes.POINTERDOWN && pointerInfo.event.buttons & 1 && this.getParameters().interactionEnabled()) {
```

Comment:
Will be more readable if you don't use logical &. If possible :-)

Reply:
Done! You are absolutely right, if any other button is pressed then we shouldn't activate the tool anyway.

Reply:
I'm still using logical & in textureCanvasManager because I think user should be able to press MMB + LMB and pan, (using === would mean they can only press one or the other) but it's prob. not that important either way.

Reply:
Sure, where it makes sense (when 2 buttons pressed at the same time) keep it in :-)